### PR TITLE
[BLADE-27] - Establish nightly builds

### PR DESCRIPTION
**Pull request by bane**

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
 
   e2eibft:
     name: IBFT E2E Tests
```
Comment (Stefan-Ethernal) [inline .github/workflows/nightly.yml]:
```suggestion
    name: Legacy E2E Tests
```

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
 
   e2eibft:
```
Comment (Stefan-Ethernal) [inline .github/workflows/nightly.yml]: I'd consider renaming this as well to legacy-e2e (or something like that).

File: .github/workflows/deploy.testnet.yml [hunk 3]
```diff
@@ -1,180 +0,0 @@
----
-name: TestNet Workflow
```
Comment (Stefan-Ethernal) [inline .github/workflows/deploy.testnet.yml:2]: Ultimately, we should consider reviving this workflow, which would spin up a testnet with the new software version and perform a set of sanity check tests that you implemented on each release or even on-demand.

File: .github/workflows/deploy.devnet.yml [hunk 4]
```diff
@@ -1,190 +0,0 @@
----
-name: DevNet Workflow
```
Comment (Stefan-Ethernal) [inline .github/workflows/deploy.devnet.yml:2]: We could probably use this workflow to spin up a new devnet on demand on AWS and then we could perform automatic or even manual testing against it. WDYT?

**Comment by bane** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by bane** [comment]
recheck

**Review by Stefan-Ethernal — COMMENTED** [review]
I'd consider to ultimately have both `deploy.devnet.yml` and `deploy.testnet.yml` workflows (if not through this PR, then through another one in the future). 

Those two would be similar in a way that both spin up a network, but with one difference
- `deploy.devnet.yml` deploys a network from scratch (running the entire bootstrap workflow)
- `deploy.testnet.yml` would be bootstrapped manually for the first time and then it would just deploy the certain commit, branch, or some release candidate and reboot the nodes.

File: .github/workflows/deploy.nightly.devnet.yml [hunk 5]
```diff
@@ -59,8 +71,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
-          repository: maticnetwork/terraform-polygon-supernets
-          ref: main
+          repository: Ethernal-Tech/blade-deployment
+          ref: refactoring
```
Comment (visnja) [inline .github/workflows/deploy.nightly.devnet.yml:75]: Don't forget to update this to main when the refactoring is done

File: .github/workflows/deploy.nightly.devnet.yml [hunk 6]
```diff
@@ -263,8 +184,8 @@ jobs:
...
```
Comment (visnja) [inline .github/workflows/deploy.nightly.devnet.yml:188]: The same goes here, change to `main` once refactoring is complete

**Review by Stefan-Ethernal — DISMISSED** [review]

**Review by Stefan-Ethernal — APPROVED** [review]

**Pull request merged**